examples/AspNetCore/README.md

```
@@ -0,0 +1,207 @@
+## Quick Start
+
+### 1. Start the Application
+
+```bash
+cd src
+dotnet run
+```
+
+The application will start on:
+
+- HTTPS: https://localhost:64494
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)